app/frontend/css/application.scss

```diff
@@ -101,6 +101,14 @@ $govuk-global-styles: true; // https://frontend.design-system.service.gov.uk/sas
     }
   }
 
+  &-margin-bottom-9 {
+
+    @media (max-width: 40.0625rem) {
```

Comment:
Was the 40.0625rem not set as a global (something like desktop-width)?

Reply:
updated